src/notifications/src/NotificationConsumer.ts

```diff
@@ -0,0 +1,61 @@
+import * as fs from "fs";
+import sendEmail from './index';
+import * as cassandra from 'cassandra-driver';
+import * as kafka from 'kafka-node';
+    
+const client = new kafka.KafkaClient();
+const consumer = new kafka.Consumer(
+    client,
+    [
+        { topic: 'notif', partition: 0 }
+    ],
+    {
+        autoCommit: false
+    }
+);
+
+const getBlacklisted = 'SELECT * FROM blacklist WHERE name = ?';
+const getUserID = 'SELECT user_id FROM camera_users WHERE camera_id = ?';
+const getEmail = 'SELECT email FROM users_id WHERE user_id = ?';
+const getImg = 'SELECT frame FROM cv_frames WHERE frame_id = ?';
+const storeNotif = 'INSERT INTO notif (date, type, email, name, frame_id) VALUES (%s, %s, %s, %s, %s)';
+
+consumer.on('message', async function(message:any){
+    const cassClient = new cassandra.Client({
+        contactPoints: ['127.0.0.1'],
+        localDataCenter: 'datacenter1',
+        keyspace: 'streams',
+    });
+    handleNotif(message, cassClient);
+})
+
+export default async function handleNotif(message: any, cassClient: cassandra.Client) {

```

Comment:
message: kafka.Message ?

Reply:
I can change it in the consumer.on function but here if I were to change it, it would give me an error regarding assignment types

Reply:
hmm ok